version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
```

Comment:
I've definitely observed conversations in various online fora where atheism is *not* so assumed. In fact, atheism is what all the fundamentalists can agree among themselves to gang up on as a shared mutual threat.